### PR TITLE
Start branch v1.x

### PR DESCRIPTION
**Pull request by tshemsedinov**
This PR will start new `v1.x`, remove config and options support to simplify package structure, temporary remove schema serialization, but we need it to start moving library to the new milestones of metaserverless. @nechaido

### CODE REVIEW

**Comment by tshemsedinov** [comment]
This PR will start new `v1.x`, remove config and options support to simplify package structure, temporary remove schema serialization, but we need it to start moving library to the new milestones of metaserverless. @nechaido

File: test/validate.js [hunk 1]
```diff
@@ -1,203 +0,0 @@
-'use strict';
```
Comment (nechaido) [inline test/validate.js]: Why is this file deleted?

File: lib/schema.js [hunk 2]
```diff
@@ -96,28 +134,13 @@ class Metaschema {
...
-      }
-
-      if (processors.preprocess) {
-        for (const preprocessor of processors.preprocess) {
-          errors.push(...preprocessor(schema));
-        }
-      }
-      if (processors.validateSchema) {
-        for (const validator of processors.validateSchema) {
-          errors.push(...validator(schema));
```
Comment (nechaido) [inline lib/schema.js]: Shouldn't we validate schemas?

File: .prettierrc [hunk 3]
```diff
@@ -5,10 +5,6 @@
     {
       "files": [".prettierrc", ".metadocrc"],
       "options": { "parser": "json" }
-    },
-    {
-      "files": ["*.category", "*.domains"],
```
[the remark this answers is not in the document]
Comment (tshemsedinov) [reply .prettierrc]: We have `*.category` and `*.domains` files only in tests but we need no tests in npm.

File: babel.config.js [hunk 4]
```diff
@@ -1,5 +0,0 @@
-'use strict';
```
[the remark this answers is not in the document]
Comment (tshemsedinov) [reply babel.config.js]: We may bring this functionality back after refactoring. But if we need just this config file, it's ok to preserve this file. I removed it together with babel serializer.

File: lib/schema.js (hunk 2 above)
Comment (tshemsedinov) [reply lib/schema.js]: That's strange, do not know why this is removed, maybe this is a result of rebase, I'll check.

File: .prettierrc (hunk 3 above)
[the remark this answers is not in the document]
Comment (nechaido) [reply .prettierrc]: This is not for npm, it for having consistant style throughout both code and schemas.

File: babel.config.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (nechaido) [reply babel.config.js]: We also needhttps://github.com/metarhia/metaschema/pull/247/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L26 and

File: lib/schema.js (hunk 2 above)
Comment (nechaido) [reply lib/schema.js]: This could have happened due to the fact there is no default validator and it was previously created from "system" schemas describing `Category` and `Domain` and then injected into config.

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metaschema/commit/8359087d174ff7d2bbe4f54408cc664573b19a73, https://github.com/metarhia/metaschema/commit/8d58fe5636bba003e5500b433e3a0a853b9bb263, https://github.com/metarhia/metaschema/commit/1d4aa83d4dbef4fa0ca4261b311c4d5316474fcd, https://github.com/metarhia/metaschema/commit/1d4aa83d4dbef4fa0ca4261b311c4d5316474fcd, https://github.com/metarhia/metaschema/commit/28dad1d008eb990aef5c0192066e589068790cff, https://github.com/metarhia/metaschema/commit/b558cd0e00db0a0fe8bd5ff15ce4d1d26033c48c, https://github.com/metarhia/metaschema/commit/a1281593673d0a36414e4fa8b787025ae476f5d9, https://github.com/metarhia/metaschema/commit/2733dadab5e9bb8cc4b0ed224dd847ea33ca8c5c, and https://github.com/metarhia/metaschema/commit/8b7b46ceda48b55e6c5ef70fc91ed57ac616d3ee
